src/App.js

```diff
@@ -22,7 +23,7 @@ class App extends Component {
         this.state = {
             devices: JSON.parse(cachedDevices) || [],
             inputText: '',
-            curTheme: defaultTheme,
+            curTheme: VanillaTheme,
```

Comment:
Here you can call selectThemeByName with empty parameters in order to get the default theme. VanillaTheme is not needed in this component